web-components/src/components/account/account.stories.tsx

```diff
@@ -0,0 +1,14 @@
+import AccountAddress from './AccountAddress';
+
+export default {
+  title: 'Account',
+  component: AccountAddress,
+};
+
+export const accountAddress = (): JSX.Element => (
+  <AccountAddress
+    onClick={() => {}}
+    name="Regen Network Development, PBC"
+    address="regen:1p6syuqk3e5a8hwp8e20jyjwr8p7nj270x4spqm"
+  />
```

Comment:
Should we use the latest story format and an action for the onClick parameter?
The `Dummy` story should be up to date as an example.

Reply:
good point, this is because this file wasn't just renamed, I haven't touched the underlying code but I can update the story nevertheless

Reply:
well, while updating this to CSF3, I've realized that the AccountAddress component is not actually being used anymore, should we just drop it?

Reply:
Oh, probably then yes.